getBundles.js

```diff
@@ -0,0 +1,13 @@
+const axios = require('axios');
+const fs = require('fs');
+
+axios.get('http://localhost:5000/public/bundle.js')
+  .then(res => {
+    fs.writeFile('./public/resBundle.js', res.data, (err) => {
+      if (err) {console.log('get bundle failed: ', err)};
+      console.log('reservation bundle updated')
+    })
+  })
+  .catch(err => {
+    console.log('getting bundle failed: ', err);
+  })
```

Comment:
Wondering if this fs.writeFile is necessary if we are gonna just use CDN's to grab our bundles while running our servers?

Reply:
yeah it's totally going to be refactored out.  i kept it for our meeting just in case, but writing the bundles seems like a rough extra step.

Reply:
💯